### PR TITLE
Translate basics into chinese.

### PR DESCRIPTION
**Pull request by cizixs**
This commit includes following changes:
+ add Chinese translation conf in _config.yml
+ translate index.md
+ translate all basics lessons

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you so very much @cizixs, this is fantastic!!  Are you okay with me merging (and deploying)?

**Comment by cizixs** [comment]
@doomspork . Yes, please merge and deploy this commit as you wish, and I will finished the other two sections ASAP.

**Comment by doomspork** [comment]
:boom: Thanks so much @cizixs

File: cn/lessons/basics/mix.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+---
+
+layout: page title: Mix category: basics order: 8
```
Comment (doomspork) [inline cn/lessons/basics/mix.md]: @cizixs I just noticed there some problems with the page meta data.  I pushed a few changes already.  I'm going to clean them up and make sure it all deploys.

Comment (doomspork) [inline cn/lessons/basics/mix.md]: Two issues I found in `mix.md` and `pattern-matching.md` just to be aware of:

+ Newlines were stripped from meta data
+ Metadata block must be start and end with exactly three dashes `---`

**Pull request merged**